test/config/utility.cc

```diff
@@ -392,9 +393,7 @@ void ConfigHelper::initializeTls(envoy::api::v2::auth::CommonTlsContext& common_
   auto* validation_context = common_tls_context.mutable_validation_context();
   validation_context->mutable_trusted_ca()->set_filename(
       TestEnvironment::runfilesPath("test/config/integration/certs/cacert.pem"));
-  validation_context->add_verify_certificate_hash(
-      "E0:F3:C8:CE:5E:2E:A3:05:F0:70:1F:F5:12:E3:6E:2E:"
-      "97:92:82:84:A2:28:BC:F7:73:32:D3:39:30:A1:B6:FD");
+  validation_context->add_verify_certificate_hash(TEST_CLIENT_CERT_HASH);
```

Comment:
I'm not sure how I really feel about it, but it seems that you might also `#define` file path, to keep those 2 values together?

Reply:
Yeah, the downside of doing this is that would be non-local during cert generation, e.g. you would then need to know your relative path to the repo root. The main thing is we catch skew between generated outputs and the locations they are consumed, the paths of the generated outputs are more stable I think.